src/views/AgmExtension.vue

```diff
@@ -260,6 +272,11 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin, FilingM
 
   /** Called when component is created. */
   created (): void {
+    if (!IsAuthorized(AuthorizedActions.AGM_EXTENSION_FILING)) {
+      // user is not authorized to access AGM extensions, so route to dashboard
+      this.authErrorDialog = true
+      throw new Error('You are not authorized to complete this action.')
```

Comment:
Do we need both a dialog and a console error? Could you just return here to pre-empt the rest of created()?